src/library/themes/themeFromTokens.js

```diff
@@ -26,27 +26,29 @@ const themeRampMap = Object.keys(colorAliases)
   }, {});
 
 export default function themeFromTokens({ themeRamp, tokens }: Args): Theme {
-  return Object.keys(tokens).reduce((acc, key) => {
-    const isBrandKey = contains(key, 'brand');
-    const value = tokens[key];
-
-    const newKey = isBrandKey ? key.replace('brand', 'theme') : key;
-    let newValue = value;
-
-    if (typeof value === 'string') {
-      if (isBrandKey) {
-        newValue = (themeRamp && themeRamp[themeRampMap[key]]) || tokens[key];
-      } else if (contains(key, 'fontSize')) {
-        newValue = remToEm(value);
-      } else if (
-        value.split('px').length === 2 &&
-        !contains(key, 'breakpoint')
-      ) {
-        newValue = pxToEm(value);
+  return Object.keys(tokens)
+    .sort()
```

Comment:
themeFromTokens is called in createTheme, which we need to keep as performant as possible. We should move this to groupedMineralTheme and the new tokens page, where it is needed.